roles/core/nic/templates/ifcfg.j2

```diff
@@ -1,53 +1,59 @@
+#jinja2: lstrip_blocks: "True"
 #### Blue Banquise file ####
 ## {{ansible_managed}}
 
 {# 1. First the name and default parameters. #}
 NAME={{item}}
 DEVICE={{item}}
-{% if (network_interfaces[item]['autoconnect'] is defined and not none) and (network_interfaces[item]['autoconnect'] == false) %}
+{% if (network_interfaces[item]['autoconnect'] is defined and network_interfaces[item]['autoconnect'] is not none) and (network_interfaces[item]['autoconnect'] == false) %}
 ONBOOT=no
 {% else %}
 ONBOOT=yes
 {% endif %}
+{% if ansible_facts.distribution_major_version < 8 %}
```

Comment:
```suggestion
{% if ansible_facts.distribution_major_version|int < 8 %}
```

Otherwise templating fails with: '<' not supported between instances of 'AnsibleUnsafeText' and 'int'